src/dist.config.ts

```diff
@@ -170,10 +170,19 @@ function webpackConfig(args: any): webpack.Configuration {
 	if (Array.isArray(args.compression)) {
 		args.compression.forEach((algorithm: 'brotli' | 'gzip') => {
 			config.plugins!.push(
-				new CompressionPlugin({
-					algorithm: algorithm === 'brotli' ? 'brotliCompress' : 'gzip',
-					test: /\.(js|css|html|svg)$/
-				})
+				new CompressionPlugin(
+					algorithm === 'brotli'
+						? {
+								filename: '[path].br[query]',
+								algorithm: 'brotliCompress',
+								test: /\.(js|css|html|svg)$/,
+								cache: false
+						  }
+						: {
+								test: /\.(js|css|html|svg)$/,
+								cache: false
+						  }
+				)
```

Comment:
Changing plugin options always invalidate cache

Reply:
I've added a comment on the other issue